source/ITimer.h

```diff
@@ -0,0 +1,89 @@
+/*
```

Comment:
We use lgpl license in rialto-gstreamer not apache

Reply:
changed